### DISCUSSION TITLE
Unable to access raw files from github

### DISCUSSION THREAD

**Discussion by harshdeepkanhai** [community/community#40451] [Other Feature Feedback, Questions, & Ideas]
I am unable to access https://raw.githubusercontent.com/TheSpeedX/PROXY-List/master/http.txt from my laptop. Can you help me resolve this issue? I can fetch it from my phone but not from my laptop.

**Comment by airtower-luna** [comment]
That sounds like there might be a problem with your laptop or the network it's on, but I'm afraid you'll have to provide some more detail to get a useful answer. A good problem description should include at least:

* What exactly are you doing?
* What happens (including full error messages, if any)?
* What would you expect to happen instead?

Try to be as precise as possible.

**Reply by devarsh-mavani-19** [reply]
Same issue. 
I am trying to open raw files of public repo but it always gets connection timeout. I tried it on different device and it works but does not work on my linux pc on same network. I tried clearing cache and cookies but doesnt work. @harshdeepkanhai did u find any solution

**Reply by harshdeepkanhai** [reply]
No, I didn't find any solution I tried flushing my DNS but that still doesn't work.
